### PR TITLE
[vi] Unsupervised Learning

### PR DESCRIPTION
**Pull request by monsterpooka**
translated to line 18

### CODE REVIEW

**Comment by shervinea** [comment]
Thank you for working on it @monsterpooka!

File: vi/cs-229-unsupervised-learning.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+
+**14. k-means clustering**
+
+&#10230; Phân cụm K-Means
+
+<br>
+
+**15. We note c(i) the cluster of data point i and μj the center of cluster j.**
+
+&#10230; Chúng ta chú ý c(i) là cụm của điểm dữ liệu i và μj là chính giữa của j.
```
Comment (damminhtien) [inline vi/cs-229-unsupervised-learning.md]:
```suggestion
&#10230; Chúng ta chú ý c(i) là cụm của điểm dữ liệu i và μj là tâm của cụm j.
```

File: vi/cs-229-unsupervised-learning.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+
+**22. [Ward linkage, Average linkage, Complete linkage]**
+
+&#10230; [Liên kết nhóm, Liên kết trung bình, Liên kết hoàn chỉnh]
+
+<br>
+
+**23. [Minimize within cluster distance, Minimize average distance between cluster pairs, Minimize maximum distance of between cluster pairs]**
+
+&#10230; [Tối thiểu hóa khoảng cách giữa các cụm, tối thiếu hiểu hóa khoảng cách trung bình giữa các cặp cụm, Tối thiểu hóa khoảng cách tối đa giữa các cặp cụm]
```
Comment (damminhtien) [inline vi/cs-229-unsupervised-learning.md]:
```suggestion
&#10230; [Tối thiểu hóa khoảng cách giữa các cụm, tối thiểu hóa khoảng cách trung bình giữa các cặp cụm, Tối thiểu hóa khoảng cách tối đa giữa các cặp cụm]
```

File: vi/cs-229-unsupervised-learning.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+
+**24. Clustering assessment metrics**
+
+&#10230; Ma trận đánh giá phân cụm
+
+<br>
+
+**25. In an unsupervised learning setting, it is often hard to assess the performance of a model since we don't have the ground truth labels as was the case in the supervised learning setting.**
+
+&#10230; Trong một cài đặt học không giám sát, thường khó để đánh giá hiệu quả của một mô hình bở chúng ta không có tập dữ liệu đã gán nhán chuẩn như trong trường hợp cài đặt học có giám sát.
```
Comment (damminhtien) [inline vi/cs-229-unsupervised-learning.md]:
```suggestion
&#10230; Trong khi thiết lập học không giám sát, thường khó để đánh giá hiệu quả của một mô hình bở chúng ta không có tập dữ liệu đã gán nhán chuẩn như trong trường hợp cài đặt học có giám sát. 
```

File: vi/cs-229-unsupervised-learning.md [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+
+**25. In an unsupervised learning setting, it is often hard to assess the performance of a model since we don't have the ground truth labels as was the case in the supervised learning setting.**
+
+&#10230; Trong một cài đặt học không giám sát, thường khó để đánh giá hiệu quả của một mô hình bở chúng ta không có tập dữ liệu đã gán nhán chuẩn như trong trường hợp cài đặt học có giám sát. 
+
+<br>
+
+**26. Silhouette coefficient ― By noting a and b the mean distance between a sample and all other points in the same class, and between a sample and all other points in the next nearest cluster, the silhouette coefficient s for a single sample is defined as follows:**
+
+&#10230; Hệ số hình chiéu - Với a là khoảng cách trung bình giữa một mẫu với tất các điểm khác trong cùng nhóm, và b là khoảng cách trung bình giữa một mẫu với tất cacr các điểm khác trong cụm gần nhất, hệ số hình chiếu s cho một mẫu đơn được định nghĩa như sau:
```
Comment (damminhtien) [inline vi/cs-229-unsupervised-learning.md]:
```suggestion
&#10230; Hệ số hình chiếu - Với a là khoảng cách trung bình giữa một mẫu với tất các điểm khác trong cùng nhóm, và b là khoảng cách trung bình giữa một mẫu với tất cả các điểm khác trong cụm gần nhất, hệ số hình chiếu s cho một mẫu đơn được định nghĩa như sau: 
```

File: vi/cs-229-unsupervised-learning.md [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+
+**26. Silhouette coefficient ― By noting a and b the mean distance between a sample and all other points in the same class, and between a sample and all other points in the next nearest cluster, the silhouette coefficient s for a single sample is defined as follows:**
+
+&#10230; Hệ số hình chiéu - Với a là khoảng cách trung bình giữa một mẫu với tất các điểm khác trong cùng nhóm, và b là khoảng cách trung bình giữa một mẫu với tất cacr các điểm khác trong cụm gần nhất, hệ số hình chiếu s cho một mẫu đơn được định nghĩa như sau: 
+
+<br>
+
+**27. Calinski-Harabaz index ― By noting k the number of clusters, Bk and Wk the between and within-clustering dispersion matrices respectively defined as**
+
+&#10230; Chỉ mục Calinski-Harabaz - Với k là số cụm, Bk và Wk là ................ được định nghĩa bởi
```
Comment (damminhtien) [inline vi/cs-229-unsupervised-learning.md]:
```suggestion
&#10230; Chỉ mục Calinski-Harabaz - Với k là số cụm, Bk và Wk là các ma trận phân tán ở giữa và bên trong cụm được định nghĩa tương đối như
```

**Comment by shervinea** [comment]
A PR on the same content is currently at the review stage in #199. Please feel free to directly add your comments and incorporate the work of this PR there!

**Pull request closed without merging**